FirstAidMedicine/ViewModel/AuthViewModel.swift

```diff
@@ -29,4 +29,43 @@ final class AuthViewModel: ObservableObject {
     // ! 논리부정연산자 true를 false로 false를 true 로  변환하다.
     //Auth.auth().currentUser가 nil이 아닌 경우 .loggedIn으로, nil인 경우 .loggedOut으로 초기화됩니다.
     //currentUser 현재 사용자의 고유식별자를 가져온다.
+    
+    func registerUser(withEmail email: String, password: String, fullName: String) async throws {
+        let result = try await Auth.auth().createUser(withEmail : email, password: password)
+        let user = User (fullName : fullName, email : email, uid : result.user.uid )
+        try await storeUser(with : user)
+        loginState = .loggedIn
+    }
+    
+    func signInUser(withEmail email : String, password : String, fullName: String) async throws {
+        try await Auth.auth().signIn(withEmail : email, password : password)
+        loginState = .loggedIn
+    }
```

Comment:
```suggestion
    func signInUser(withEmail email : String, password : String) async throws {
        try await Auth.auth().signIn(withEmail : email, password : password)
        loginState = .loggedIn
    }
```

오늘 같이 봤던 내용처럼 요기서 fullName을 매개변수로 받아줄 필요 없을 것 같습니다!